src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts

```diff
@@ -58,7 +60,9 @@ export class CodeEditorTutorAssessmentInlineFeedbackComponent {
     faTrashAlt = faTrashAlt;
     faExclamationTriangle = faExclamationTriangle;
 
-    constructor(private translateService: TranslateService, public structuredGradingCriterionService: StructuredGradingCriterionService) {}
+    constructor(private translateService: TranslateService, public structuredGradingCriterionService: StructuredGradingCriterionService, elementRef: ElementRef) {
+        this.elementRef = elementRef;
+    }
```

Comment:
This should also work instead of adding an attribute above.
```suggestion
    constructor(private translateService: TranslateService, public structuredGradingCriterionService: StructuredGradingCriterionService, public elementRef: ElementRef) {
    }
```

Reply:
I tried that at first, but then IntelliJ complained about an "unused variable" and I didn't want to suppress the warning if I could go with the other solution. Thoughts?

Reply:
<img width="883" alt="screenshot-2023-07-08_001513" src="https://github.com/ls1intum/Artemis/assets/9006596/b4e9f4c9-99a5-4a7a-8bc1-e2d0ef2cfe83">

IntelliJ is wrong here (the suggestion works like this), but I fear that someone else will try to remove this in the future because of this wrong warning.